conn.go

```diff
@@ -39,6 +39,8 @@ type Conn struct {
 	reqs          sync.Map
 	cancelSession context.CancelFunc
 	sessionCtx    context.Context
+
+	writeRecordsChan chan *writeRecordRequest
```

Comment:
what do you think of giving it a shorter name, like "wchan" or something

Reply:
as you wish, I like to write code as close to English as possible to alleviate the cognitive burden when I read the code after six months.